### PR TITLE
Add powershell custom prompt Script

### LINKED ISSUE #633: Add PowerShell Prompt customization script
Customize your PowerShell Prompt with git status, execution time, system info, etc

### PR DESCRIPTION
**Pull request by avinal**
# Description

PowerShell prompt script to show more information like git branch status, command execution time, current directory, admin status, etc.

Fixes #633 

Replace `issue_no` with the issue number which is fixed in this PR

## Type of change

_Please delete options that are not relevant._

- [x] New feature (non-breaking change which adds functionality)
- [x] This change requires a documentation update

# Checklist:

- [x] My code follows the style guidelines(Clean Code) of this project
- [x] I have performed a self-review of my own code
- [x] I have commented my code, particularly in hard-to-understand areas
- [x] I have created a helpful and easy to understand `README.md`

### CODE REVIEW

**Review by Programmer1473 — APPROVED** [review]
good:)

File: PowerShell/Custom_Prompt/profile.ps1 [hunk 1]
```diff
@@ -0,0 +1,42 @@
...
+
+    if ($branch -ne $NULL) {
+        # we're on an actual branch, so print it
+        Write-Host "($branch)" -ForegroundColor Green
+    } else {
+        Write-Host ""
+    }
+
+    return "> "
+}
```
Comment (HarshCasper) [inline PowerShell/Custom_Prompt/profile.ps1:42]: Add a new line at the EOF

File: PowerShell/Custom_Prompt/README.md [hunk 2]
```diff
@@ -0,0 +1,32 @@
...
+3. Check if there is an existing configuration, if not create a new file.
+
+```powershell
+    if (!(Test-Path -Path $PROFILE)){ New-Item -Path $PROFILE -ItemType File } ; notepad $PROFILE
+```
+
+4. A file will open, if there is an exisiting `prompt` function in the file then change the contents else paste everything from [profile.ps1](profile.ps1) in the file.
+
+5. Save and exit Notepad and relaunch PowerShell. 
+
```
Comment (HarshCasper) [inline PowerShell/Custom_Prompt/README.md:28]: Can you add a few screenshots to display the output? 

You can upload them on Imgur and reference them here.

**Review by HarshCasper — CHANGES_REQUESTED** [review]
Tiny Changes. The Script looks good!

File: PowerShell/Custom_Prompt/profile.ps1 (hunk 1 above)
Comment (avinal) [reply PowerShell/Custom_Prompt/profile.ps1:42]: ok

File: PowerShell/Custom_Prompt/README.md (hunk 2 above)
Comment (avinal) [reply PowerShell/Custom_Prompt/README.md:28]: yeah sure

**Review by HarshCasper — APPROVED** [review]

**Pull request merged**